lab5/commands/Exit.java

```diff
@@ -0,0 +1,15 @@
+package org.example.commands;
+
+import org.example.utils.CollectionManager;
+
+public class Exit implements Commands{
+    public void execute() {
+        System.out.println("PROGRAM CLOSING...");
+        System.exit(-1);

```

Comment:
don't do system exit, return a flag that tells the program to stop running without using system exit

Reply:
You still have system exit left here